src/workflows/import-content.ts

```diff
@@ -18,7 +15,7 @@ const {
 });
 
 export type ImportContentOutput = {
-  preprintPath: string,
+  preprintPath?: string,
```

Comment:
I would probably just remove this - it was helpful only to report/diagnose an action we're no longer taking (finding the path from biorxiv)